x/muxer/tests/muxer_test.go

```diff
@@ -68,6 +69,20 @@ func TestSimpleDispatch(t *testing.T) {
 	)
 }
 
+func TestCheckConfigErrorNotDuplicated(t *testing.T) {
+	var m muxer.DispatchTable
+	m.Resources = map[string]int{
+		"test:mod:A": 0,
+		"test:mod:B": 1,
+	}
+	req := `{}`
+	mux(t, m).replay(
```

Comment:
I would love to see a test that confirms that multiple separate errors are not combined or lost.

Reply:
I updated the tests with a new one for different errors, is that what you meant?

Reply:
Yes! Thanks!